package.json

```diff
@@ -17,6 +17,7 @@
 		"builtin-modules": "^3.2.0",
 		"esbuild": "0.13.12",
 		"obsidian": "^0.12.17",
+		"process": "^0.11.10",
```

Comment:
Please explain what this does

Reply:
This gets imported in the esbuild file, but isn't in the dev dependencies. It doesn't have any real impact, it just makes it more explicit.